frontend/src/tests/components/CollegiateSubreddits/CollegiateSubredditsTable.test.js

```diff
@@ -0,0 +1,157 @@
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import { currentUserFixtures } from "fixtures/currentUserFixtures";
+
+const mockedNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useNavigate: () => mockedNavigate
+}));
+
+const mockedMutate = jest.fn();
+
+jest.mock('main/utils/useBackend', () => ({
+    ...jest.requireActual('main/utils/useBackend'),
+    useBackendMutation: () => ({mutate: mockedMutate})
+}));
+
+describe("UserTable tests", () => {
+    const queryClient = new QueryClient();
+  
+  
+    test("renders without crashing for empty table with user not logged in", () => {
+      const currentUser = null;
+  
+      render(
+        <QueryClientProvider client={queryClient}>
+          <MemoryRouter>
+            <CollegiateSubredditsTable subreddits={[]} currentUser={currentUser} />
+          </MemoryRouter>
+        </QueryClientProvider>
+  
+      );
+    });
+    test("renders without crashing for empty table for ordinary user", () => {
+      const currentUser = currentUserFixtures.userOnly;
+  
+      render(
+        <QueryClientProvider client={queryClient}>
+          <MemoryRouter>
+            <CollegiateSubredditsTable subreddits={[]} currentUser={currentUser} />
+          </MemoryRouter>
+        </QueryClientProvider>
+  
+      );
+    });
+  
+    test("renders without crashing for empty table for admin", () => {
+      const currentUser = currentUserFixtures.adminUser;
+  
+      render(
+        <QueryClientProvider client={queryClient}>
+          <MemoryRouter>
+            <CollegiateSubredditsTable subreddits={[]} currentUser={currentUser} />
+          </MemoryRouter>
+        </QueryClientProvider>
+  
+      );
+    });
+  
+    test("Has the expected colum headers and content for adminUser", () => {
+  
+      const currentUser = currentUserFixtures.adminUser;
+  
+      const { getByText, getByTestId } = render(
+        <QueryClientProvider client={queryClient}>
+          <MemoryRouter>
+            <CollegiateSubredditsTable subreddits={collegiateSubredditsFixtures.threeSubreddits} currentUser={currentUser} />
+          </MemoryRouter>
+        </QueryClientProvider>
+  
+      );
+  
+      const expectedHeaders = ["id","name","location","subreddit"];
+      const expectedFields = ["id","name","location","subreddit"];
+      const testId = "CollegiateSubredditsTable";
+  
+      expectedHeaders.forEach((headerText) => {
+        const header = getByText(headerText);
+        expect(header).toBeInTheDocument();
+      });
+  
+      expectedFields.forEach((field) => {
+        const header = getByTestId(`${testId}-cell-row-0-col-${field}`);
+        expect(header).toBeInTheDocument();
+      });
+  
+      expect(getByTestId(`${testId}-cell-row-0-col-id`)).toHaveTextContent("1");
+      expect(getByTestId(`${testId}-cell-row-1-col-id`)).toHaveTextContent("2");
+  
```

Comment:
Why do you have 3 collegiate subreddits in your fixtures, load up 3 collegiate subreddits into the table, and then only check 1 and 2???

Add this on line 93:
expect(getByTestId(`${testId}-cell-row-1-col-id`)).toHaveTextContent("3");

Reply:
Good catch, I pushed the correction to the test now